src/include/nfs/ui.rb

```diff
@@ -51,9 +69,12 @@ def initialize_nfs_ui(include_target)
         # Help, part 2 of 4
         _(
           "<p>Each NFS share is identified by remote NFS server address and\n" \
```

Comment:
I'm not sure, but maybe the " and" in this line could be replaced by ",". What do you think?

Reply:
To be honest, I am not sure. The couple "server" and "directory" are very attached, and they identify the device. I would keep it as it was (I haven't modified it).